src/OpenSSL/SSL.py

```diff
@@ -130,6 +132,19 @@ class _buffer(object):
 SSL_CB_HANDSHAKE_START = _lib.SSL_CB_HANDSHAKE_START
 SSL_CB_HANDSHAKE_DONE = _lib.SSL_CB_HANDSHAKE_DONE
 
+# Taken from https://golang.org/src/crypto/x509/root_linux.go
```

Comment:
We won't ship a precompiled binary for those platforms so we shouldn't need to care.